models/channel_helpers.go

```diff
@@ -90,6 +90,23 @@ func checkWebHookConfig(c *WebHookConfig) error {
 		return status.Error(codes.InvalidArgument, "Webhook url field is empty.")
 	}
 
+	if c.HTTPConfig != nil {
+		if tlsConfig := c.HTTPConfig.TLSConfig; tlsConfig != nil {
+			if tlsConfig.CAFile != "" && tlsConfig.CAFileContent != "" {
```

Comment:
question: Shouldn't be provided at least one of them, or it doesn't matter and can be empty?

Reply:
the rule is they should not be set at the same time, both could be empty.